src/main/java/com/epam/prejap/teatrees/block/Color.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.prejap.teatrees.block;
+
+/**
+ * Class with predefined color constants, which include unique ANSI code
```

Comment:
```suggestion
 * Predefined colours, which include unique ANSI code
```

First of all enum not a class. Secondly, we can see what it is just below. We are after what it does not what it is.